Website/src/rendering/src/assets/css/components/Speakers/SpeakersGrid.css

```diff
@@ -1,59 +1,38 @@
-.section__speakers__grid {
-  @apply mx-auto grid grid-cols-1 gap-5;
-  @apply sm:grid-cols-2;
-  @apply md:grid-cols-3;
-  @apply lg:grid-cols-4;
-  @apply xl:grid-cols-5;
-
-  margin-top: 20px;
-  margin-bottom: 40px;
-
-  &__speaker {
-    @apply box-border col-span-1 relative;
-
-    width: 100px;
-    min-width: 260px;
-    padding: 10px;
-    margin: 10px;
-
-    &:hover img {
-      box-shadow: 0 0 10px rgba(23, 23, 23, 0.45);
-    }
-
-    img {
-      @apply w-full;
-    }
+.section__speakers {
+  .section__content__title {
+    @apply text-center text-blue normal-case;
+  }
 
-    .speaker__image {
-      @apply w-full bg-none;
-    }
+  .section__speakers__grid {
+    @apply mx-auto grid grid-cols-1 gap-5 mt-5 mb-10;
+    @apply sm:grid-cols-2;
+    @apply md:grid-cols-3;
+    @apply xl:grid-cols-4 gap-10;
 
-    .speaker__featured {
-      @apply absolute top-2 right-1 text-yellow;
+    &__speaker {
+      @apply box-border col-span-1 relative;
 
-      .icon {
-        @apply h-4 mr-2 inline text-yellow;
+      &:hover img {
+        box-shadow: 0 0 10px rgba(23, 23, 23, 0.45);
       }
 
-      svg {
-        filter: drop-shadow(1px 1px 2px black);
+      img {
+        @apply w-full;
       }
-    }
 
-    .speaker__name {
-      @apply mx-auto text-center font-semibold text-black;
+      .speaker__image {
+        @apply w-full bg-no-repeat bg-cover bg-blue;
 
-      margin-top: 5px;
-      margin-bottom: 5px;
-    }
+        background-image: url(../../../img/background-element.svg);
+      }
 
-    .speaker__role {
-      @apply mx-auto text-center font-medium uppercase text-blue;
+      .speaker__name {
+        @apply mx-auto text-left font-bold text-black my-1;
```

Comment:
Isn't `mx-auto` and `text-left` doing opposite things? Do we want the text centered or left aligned here?